azure-pipelines.yml

```diff
@@ -41,15 +41,26 @@ steps:
     dockerHubImageName: $(imageName)
     CC_TEST_REPORTER_ID: $(ccReporterID)
 - script: |
+    set -x
+    set e
```

Comment:
I meant `set -e`, but we don't want this now since we need to ensure that the `docker cp` always happens. `set -e` would stop the script from running as soon as `parallel:spec` returned an error.